src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -115,18 +117,39 @@ def _get_payload(self, message: Message[KafkaPayload]) -> MetricsBucket:
         payload = json.loads(message.payload.value.decode("utf-8"), use_rapid_json=True)
         return cast(MetricsBucket, payload)
 
-    def _count_processed_transactions(self, bucket_payload: MetricsBucket) -> int:
+    def _count_processed_items(self, bucket_payload: MetricsBucket) -> Mapping[DataCategory, int]:
         if bucket_payload["metric_id"] != self.metric_id:
-            return 0
+            return {}
         value = bucket_payload["value"]
         try:
-            return len(value)
+            quantity = len(value)
         except TypeError:
             # Unexpected value type for this metric ID, skip.
-            return 0
+            return {}
+
+        items = {DataCategory.TRANSACTION: quantity}
+
+        if bucket_payload["tags"].get(self.profile_tag) == "true":
```

Comment:
for the sake of consistency i would use `resolve_tag_value` like everywhere else. it might be that we want to mass-refactor this sort of code in the future, and being able to find all code that deals with query results by searching for `resolve` calls definitely would help there